### PR TITLE
set initial site style

### PR DESCRIPTION
**Pull request by mike-allaway**
Set the initial welcome/about page templates (with some sample content) and the overal website style

### CODE REVIEW

**Comment by mike-allaway** [comment]
This is what the site style looks like:

Screenshot 2023-06-21 at 15 54 38

File: django/.pa11yci [hunk 1]
```diff
@@ -3,16 +3,15 @@
         "runners": ["axe", "htmlcs"],
         "chromeLaunchConfig": {
             "args": ["--no-sandbox"]
-        }
+        },
+        "hideElements": [
```
Comment (ceryslewis) [inline django/.pa11yci]: What you have here will work, but is it maybe worth considering alternatives such as allowing testing within the hidden elements by either using 
```
       "ignore": [
          "color-contrast"
        ]
```
or using sed to manipulate the css to hide the background image (using the fallback colour only) in .gitlab-ci.yml before pa11y-ci is run?

**Review by ceryslewis — APPROVED** [review]
Look good. Have added a comment around an potential but optional way to enhace the accuracy of the pa11yci test

File: django/.pa11yci (hunk 1 above)
Comment (mike-allaway) [reply django/.pa11yci]: thanks, that's helpful. have added the ignore for color contrast. it passed Wave and the color.review said it was suitable but no matter what I set it to (even black) pa11y still complained. it's becoming quite a frustrating tool to use these days! thanks again for the tip

**Pull request merged**